frontend/src/sdk/commonFetch.ts

```diff
@@ -48,7 +48,8 @@ export const validateStatus = async (response: Response) => {
 
 export const commonFetch =
   (auth: AuthConfig) =>
-  async (url: string, { pathPrefix, ...options }: RequestInit & { pathPrefix?: string }): Promise<Response> => {
+  async (url: string, requestInit: RequestInit & { pathPrefix?: string } = {}): Promise<Response> => {
```

Comment:
Just curious, since I am fairly new to TS, is this just so the function definition is easier to read? Or does it add anything extra? Because I tend to avoid using extra constants just to spread them later on. if the `requestInit` would have been used later I'd be all fine with it. But I am new to TS, so this could be something benefical actually.

Reply:
I made changes here in order to resolve a typescript error with the consumer of this function. I'll investigate again.

Reply:
Looks like adding just the default value was sufficient enough. Updated.